it-IT.Rmd

```diff
@@ -1,5 +1,109 @@
 ---
-title: "`r emo::ji('Italy')` Project-wide Code of Conduct statement for Bioconductor (Italian, Italy)"
+title: "`r emo::ji('Italy')` Project-wide Codice di Buon Condotta per Bioconductor (Italiano, Italia)"
```

Comment:
> Italiano, Italia
Hm.. good point. In the other languages, I've kept that in English. It does make sense to translate that bit too. I would have to notify the maintainers of other languages to update their respective page though, which is something that I've been wondering how to set up.

Reply:
Yeah, I guess i makes sense to have it all the same way (i.e., translated) everywhere. It emphasises the internationalisation when people see all the different alphabets and languages on pages like "About". It's not a bad thing to avoid the ubiquitous English sometimes.